### DISCUSSION TITLE
模板需求区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#1] [模板需求区]
有制作模板需求的，请提交在这里。

格式如下
```
- 名称：APP或者网站名称
- 类型：APP或者网站
- 网址：APP或者网站地址
- 相关：是否有相关脚本或模板以供参考，如有，请提供相关地址
- 描述：功能需求的简单描述
```
> PS：不提供阿里系，腾讯系，银行类，支付工具，小程序,PT和机场类的模板制作;
> 模板增加并验证成功后请自行删除请求

**Comment by nnnjokernnn** [comment]
- 名称：村花论坛
- 类型：web
- 网址：https://cunhua.beauty/
- 相关：无
- 描述：签到功能无需选择字段登录不需验证码

**Comment by bgaidu** [comment]
名称：哈弗智家
    类型：APP
    网址：https://www.haval.com.cn/general/download/app/
    相关：我抓不到包，APP好像有抓包检测
    描述：每日签到，和文章点赞 给积分

**Comment by colin-nian** [comment]
- 名称：MyBMW
- 类型：APP
- 网址：[APP](https://apps.apple.com/us/app/my-bmw/id1519457734)
- 相关：抓不到包
- 描述：每日签到

**Comment by ntbowen** [comment]
- 名称：Lynk&Co领克-不止于车
- 类型：APP
- 网址：https://apps.apple.com/cn/app/lynk-co%E9%A2%86%E5%85%8B-%E4%B8%8D%E6%AD%A2%E4%BA%8E%E8%BD%A6/id1305517667
- 相关：无
- 描述：签到

**Comment by dongsee** [comment]
- 名称：CSDN
- 类型：APP
- 网址：https://www.csdn.net/
- 相关：无
- 描述：签到抽检

**Comment by Fantasy9619** [comment]
名称：夸克网盘
类型：app
网址：https://pan.quark.cn/
相关：无
描述：签到功能

**Comment by pplps** [comment]
- 名称：爱书人的港湾
- 类型：网站
- 网址：https://www.woyushu.com
- 相关：无
- 描述：每日签到
- ps：非程序员，尝试自己制作，能post账号密码登录，但是签到页面总是提示{"status":"danger","msg":"尚未登录"}，希望大佬辛苦有时间制作一个，也能借机学习一下

**Comment by shinesky** [comment]
- 名称：云中漫步
- 类型：网站
- 网址：https://www.yunzmb.com/
- 相关：无
- 描述：想要签到功能，这是一个电影动漫等各种资源下载论坛，每日签到可以获得金币，下载资源

**Comment by abu3d** [comment]
名称：比亚迪海洋
类型：APP
网址：
相关：无
描述：签到

**Reply by suhaiyou945** [reply]
希望有人能开发

**Comment by volcano6** [comment]
- 名称：新出行
- 类型：APP/网站
- 网址：https://www.xchuxing.com/
- 相关：无
- 描述：签到分享, 点赞

**Comment by gurenn** [comment]
名称：设计宝藏
类型：web
网址：https://www.shejibaozang.com/task
相关：无
描述：每日签到

**Comment by SayXieXie** [comment]
名称：普广打印论坛
类型：网站
网址：https://www.oahelp.net/
相关：https://www.oahelp.net/plugin.php?id=k_misign:sign
描述：自动签到

**Comment by NiYukk** [comment]
- 名称：漫趣社
- 类型：网站
- 网址：https://www.manqushe.com
- 相关：无
- 描述：个人中心点击签到，并显示日志

**Comment by NiYukk** [comment]
名称：中图网
类型：网站
网址：https://www.bookschina.com/
相关：无
描述：个人中心点击签到，并显示日志

**Comment by SayXieXie** [comment]
名称：办公中国
类型：网站
网址：http://www.oachn.net/
相关：无
描述：会员登录每日签到

**Comment by q505389752** [comment]
- 名称：粤漫社
- 类型：网站
- 网址：[APP或者网站地址](https://www.gomicx.net/)
- 相关：无
- 描述：每日登录网站

**Comment by kongev** [comment]
名称：老王论坛
类型：网站
网址：https://laowang.vip/
相关：签到
有滑动验证
描述：每日登录网站和签到

**Comment by kkiytb** [comment]
名称：猫修
类型：APP
网址：https://www.lmaoyx.com/
相关：无
描述：签到

**Comment by huoshenw** [comment]
名称：oai
类型：网站
网址：https://oai.itsfurry.com
相关：右上角签到
描述：账号密码已私发邮箱

**Comment by zhailala** [comment]
名称：司机社
类型：网页
网址：https://xsijishe.com
相关：每日签到可以获得车票
描述：每日签到，原来有一个，但是不行了

**Comment by zhailala** [comment]
名称：咸鱼单机
类型：网页
网址：https://www.xianyudanji.net/
描述：自动签到，其他作者有cn域名，但是cn域名已经失效了，谢谢作者！

**Comment by 404gods** [comment]
名称：serv00.com
类型：网页
网址：https://panel7.serv00.com/
描述：登陆保号，无需签到，谢谢作者

**Comment by bofpv** [comment]
名称：浙江职业足球俱乐部
类型：网站
网址：https://www.zhejiangfc1998.com/
描述：进去特定网页自动签到或点击签到
进入自动完成签到网页（[https://www.zhejiangfc1998.com/center/sign），](https://www.zhejiangfc1998.com/center/sign%EF%BC%89%EF%BC%8C)
手动：进个人中心点击签到

**Comment by bofpv** [comment]
名称：华为运动健康
类型：APP
网址：APP下载地址https://consumer.huawei.com/cn/mobileservices/health/
描述：看看能不能自动签到

**Comment by Wallemuse** [comment]
名称：盘基地资源论坛
类型：网站
网址：https://www.panjdzy.com/
相关：无
描述：登录积分，每日摇摇乐，每日可访问别人空间十次得十分

**Comment by he0075** [comment]
- 名称：棉花糖VIP-网络安全资源大全
- 类型：网站
- 网址：https://vip.bdziyi.com
- 相关：抓包未找到和签到相关的
- 描述：每日登录和签到
![image](https://github.com/user-attachments/assets/186ebd8e-4d05-49e4-8b30-e118da961e44)

**Reply by he0075** [reply]
这种个人搭建的网站，能通过此方式抓包实现自动签到吗

**Comment by xzqcsa** [comment]
- 名称：ME Frp
- 类型：网站
- 网址：https://www.mefrp.com/
- 相关：无
- 描述：进网站后在左侧“每日签到”栏中点击签到

**Reply by xzqcsa** [reply]
能帮忙写个青龙面板的自动签到脚本吗

**Comment by 00exia00** [comment]
- 名称：极核
- 类型：APP
- 网址：无
- 相关：无
- 描述：签到打卡功能

**Reply by qingyuan0o0** [reply]
Cfmoto-X-Sign 是动态签名，需要逆向 App 才能获取算法

**Comment by distress1023** [comment]
- 名称：什么值得买
- 类型：web
- 网址：https://www.smzdm.com/
- 相关：三年前大佬做过，但是失效了
- 描述：每日签到

**Comment by distress1023** [comment]
名称：98堂
类型：web
网址：https://sehuatang.net/
相关：以前大佬做过，但是失效了
描述：每日签到

**Comment by 404user** [comment]
- 名称：ZEEHO
- 类型：APP
- 网址：APP
- 相关：无
- 描述：签到功能

**Reply by qingyuan0o0** [reply]
Cfmoto-X-Sign 是动态签名，需要逆向 App 才能获取算法

**Comment by SCARS0314** [comment]
名称：MIDIFOX
类型：网站
网址：midifox.com
相关：自己学着制作了一个模板，测试后没有金币增长，但点击签到提示已签到；手动签到是能正常自动获得0.3金币的
描述：签到并获得金币

**Comment by zsq92** [comment]
名称：CFMOTO
类型：APP
网址：APP
相关：无
描述：签到打卡功能
